### ISSUE TITLE
Pano paste the latest item from the clipboard on login

### ISSUE THREAD

**Issue by MartinHeywang** [oae/gnome-shell-pano#269]
## Description

The title is quite self-explanatory : whenever I login to my session (after suspend, log out or reboot), Pano paste the most recent item from the clipboard in a focused textarea. If there is no such textarea, then the text is pasted in the "Overview" search bar.

### Problem Explanation

The bug happens whenever I log in.

### Expected Behavior

Pano should start normally without pasting any content anywhere.

### Steps To Reproduce

Steps to reproduce, if applicable:

1. Make sure Pano is activated from the Extension Manager
2. Copy some text
3. (optional) Focus on a text zone in any application
4. Suspend, log out, or reboot your computer, then log back in again
5. See the bug happening

### Details

Mark with [ ] all that applies:

#### It happens with any application?

- [x] Yes, it applies to any application.
- [ ] No. Only with the following applications:
  - VSCode
- [ ] It works with the following applications that I have tried:
  - Fill in a list with any application that applies

#### It happens only on one computer?

- [x] I don't know.
- [ ] No. I have tried it on more than one computer.

#### It happens only with some specific gnome configuration?

- [x] I don't think that the configuration matters.
- [ ] Yes. Only if the following config is set up:
  - Fill in a list with any configuration tha applies.

#### It happens only with some specific extension installed?

- [x] I don't think that the installed extensions affect the bug/behavior.
- [ ] Yes. Only if the following gnome extension is installed:
  - Fill in a list with any extension tha applies.
  - Fill in also any detail about the extensions that applies.

## Diagnostics

Fill in all information that applies:

### Environment

- GNOME Shell version: 45.5
- Distro: Fedora Linux, Workstation Edition
- Distro version: 39

#### Gnome

**Command:** `journalctl --since=now --follow /usr/bin/gnome-shell`

``` bash
$ journalctl --since=now --follow /usr/bin/gnome-shell
```
nothing in particular related to pano

#### Pano Configuration

**Command:** `dconf dump /org/gnome/shell/extensions/pano/`

``` bash
$ dconf dump /org/gnome/shell/extensions/pano/
--- file is empty, nothing printed out ---
$
```

#### Enabled Extensions

**Command:** `dconf read /org/gnome/shell/enabled-extensions | tr ' ' '\n'`

``` bash
$ dconf read /org/gnome/shell/enabled-extensions | tr ' ' '\n'
['caffeine@patapon.info',
'Hide_Activities@shay.shayel.org',
'background-logo@fedorahosted.org',
'launch-new-instance@gnome-shell-extensions.gcampax.github.com',
'rectangle@acristoffers.me',
'user-theme@gnome-shell-extensions.gcampax.github.com',
'quick-settings-tweaks@qwreey',
'logomenu@aryan_k',
'monitor@astraext.github.io',
'paperwm@paperwm.github.com',
'top-bar-organizer@julian.gse.jsts.xyz',
'gnome-ui-tune@itstime.tech',
'Battery-Health-Charging@maniacx.github.com',
'hidetopbar@mathieu.bidon.ca',
'forge@jmmaranan.com',
'trayIconsReloaded@selfmade.pl',
'quick-settings-avatar@d-go',
'pano@elhan.io']
```

**Comment by nooraibk** [comment]
I'm on fedora 39 and having same issue. Its particularly concerning because it can paste sensitive information if you have setup other apps to launch on startups.

**Comment by DAPOTTIS** [comment]
same problem here

**Comment by OsoRojo** [comment]
Same.  And it doesn't happen just on login.  It also does it resume from suspend.

**Comment by Totto16** [comment]
I could reproduce that finally, I always used x11 (because of an NVIDIA graphics card 😓 ) but under Wayland this appareas to be an issue, I'm looking into this soon ™️

**Comment by OsoRojo** [comment]
@Totto16 Lately I've tried to be better about noting the 'pure wayland' nature of my setup for exactly that reason, but I forgot when I chimed.  This is a good reminder to always note specifics like that.  Glad you were able to reproduce it!!

**Comment by Tuba2** [comment]
@MartinHeywang, is it the same issue from #244 , which was related to the "Paste on select" feature?

You can test this by disabling "Paste on select" and trying to reproduce the bug.

I believe this was patched, but I don't know for sure.

**Comment by MartinHeywang** [comment]
I don't know if it is exactly the same issue, but **the problem does disappear when I disable "Paste on select".** I have enabled and disabled the feature and they do seem correlated.

**Comment by android272** [comment]
"This issue is critical for me. I don’t always remember it happens, and Pano pastes my latest clipboard content without me noticing, inserting random text blocks into my notes and journal entries. I’m worried that if I have text selected before logging off or sleeping, it could overwrite my work, corrupting my data without my knowledge. I think this should be marked as critical since it risks data loss. Pano’s the best clipboard manager I’ve found, I don’t want to disable it, but I’ll have to until this is fixed to protect my notes.

**Comment by Totto16** [comment]
> "This issue is critical for me. I don’t always remember it happens, and Pano pastes my latest clipboard content without me noticing, inserting random text blocks into my notes and journal entries. I’m worried that if I have text selected before logging off or sleeping, it could overwrite my work, corrupting my data without my knowledge. I think this should be marked as critical since it risks data loss. Pano’s the best clipboard manager I’ve found, I don’t want to disable it, but I’ll have to until this is fixed to protect my notes.

I agree, but it is not easy to test on my side, I'll try to get a reproduction, so that i can work on a fix. 👍🏼
